sys/include/net/gnrc/ipv6/ext.h

```diff
@@ -38,6 +39,40 @@
 extern "C" {
 #endif
 
+/**
+ * @defgroup    net_gnrc_ipv6_ext_conf IPv6 extension header compile configurations
+ * @ingroup     net_gnrc_ipv6_ext
+ * @ingroup     config
+ * @{
+ */
+/**
+ * @brief   IPv6 fragmentation reassembly buffer size
+ *
+ * @note    Only applicable with [gnrc_ipv6_ext_frag](@ref net_gnrc_ipv6_ext_frag) module
+ */
+#ifndef GNRC_IPV6_EXT_FRAG_RBUF_SIZE
+#define GNRC_IPV6_EXT_FRAG_RBUF_SIZE        (1U)
```

Comment:
No, there is just one reassembly entry per node configured. I. e. The node can only receive one packet >MTU at a time. Given that fragmentation _should_ be a corner case, IPv6 already expects an MTU of at least 1280, and most of our devices only have a few kilobytes of RAM, I believe this is a reasonable default value.

Reply:
Ah I see. So the number of fragments is then dynamic / based on the amount of free heap / what's available in pktbuf_static.

Reply:
Yepp

Reply:
Ah, well no. It is upper bounded by `LIMITS_POOL_SIZE`, as the`limits` objects track the bounds of the fragments (required to prevent datagram poisoning).